src/views/CampusSafety/views/LostAndFoundAdmin/views/MissingItemList/index.tsx

```diff
@@ -137,73 +137,94 @@ const MissingItemList = () => {
+                        <InputLabel>Color</InputLabel>
+                        <Select value={color} onChange={(e) => setColor(e.target.value)}>
+                          <MenuItem value="">All</MenuItem>
+                          {colors.map((colorOption) => (
+                            <MenuItem key={colorOption} value={colorOption}>
+                              {colorOption}
+                            </MenuItem>
+                          ))}
+                        </Select>
+                      </FormControl>
+                    </Grid>
+                    <Grid item xs={isMobile}>
+                      <FormControl size="small" className={styles.formControl} fullWidth>
+                        <InputLabel>Category</InputLabel>
+                        <Select value={category} onChange={(e) => setCategory(e.target.value)}>
+                          <MenuItem value="">All</MenuItem>
+                          {categories.map((categoryOption) => (
+                            <MenuItem key={categoryOption} value={categoryOption}>
+                              {categoryOption}
+                            </MenuItem>
+                          ))}
+                        </Select>
+                      </FormControl>
+                    </Grid>
+                    <Grid item xs={isMobile}>
+                      <Button
+                        onClick={() => {
+                          setStatus('');
+                          setCategory('');
+                          setColor('');
+                          setKeywords('');
+                          handleFilter();
+                        }}
+                        variant="contained"
+                        color="error"
+                        fullWidth
+                      >
+                        Clear
+                      </Button>
+                    </Grid>
+                  </Grid>
                 </Grid>
-                <Grid item xs={isMobile}>
+                {/*Button: Report Item for others */}
+                <Grid item xs={12} sm={3} style={{ textAlign: isMobile ? 'center' : 'right' }}>
                   <Button
-                    onClick={() => {
-                      setStatus('');
-                      setCategory('');
-                      setColor('');
-                      setKeywords('');
-                      handleFilter();
-                    }}
                     variant="contained"
-                    color="error"
-                    fullWidth
+                    color="primary"
+                    onClick={() => navigate('/campussafety/LostAndFoundAdmin/ReportItemForOthers')}
+                    fullWidth={isMobile}
+                    className={styles.reportButton}
                   >
-                    Clear
+                    Report Item for others
                   </Button>
                 </Grid>
               </Grid>
```

Comment:
This is a strange visual progression...
<img width="570" alt="Screenshot 2024-11-12 at 2 41 08 PM" src="https://github.com/user-attachments/assets/62aa1e49-4ab6-4435-9a78-2a0742c71d32">
<img width="590" alt="Screenshot 2024-11-12 at 2 41 43 PM" src="https://github.com/user-attachments/assets/2fdce96b-fddf-414c-8400-6839bc8cefc7">
<img width="605" alt="Screenshot 2024-11-12 at 2 42 12 PM" src="https://github.com/user-attachments/assets/a688ca59-e03c-4508-b064-d803cbb2c3ab">
@
<img width="658" alt="Screenshot 2024-11-12 at 2 42 39 PM" src="https://github.com/user-attachments/assets/d627f828-177b-454c-b331-d9cdb2e6f325">
<img width="724" alt="Screenshot 2024-11-12 at 2 43 21 PM" src="https://github.com/user-attachments/assets/261ad2dc-2f4a-4ef7-966f-2488b5d2bec2">

Reply:
Not sure what you mean ?

Reply:
I believe he means the filter section looks very weird when window size shrinks, particularly the new "report item for others" button.  That button should probably move either on top of or under the filters on mobile, to fit better.

Reply:
it does move to the bottom for a specific size screen
<img width="383" alt="Screenshot 2024-11-12 at 9 35 27 PM" src="https://github.com/user-attachments/assets/7c120601-ea4e-4a1f-ab5c-9cf6ac24a2ed">

Reply:
its more complicated adjusting each of them for each screen size, this seemed like the best solution, that way it also doesnt affect the data view for the items

Reply:
Its on the side on screen that have room on the grid for it to fit in there and the screen that dont it adjust accordingly, untill its all on one screen size, so each of them keep shrinking kind of

Reply:
Currently it jumps around a lot as the screen changes size.  It's pretty jarring, so if we can work on the breakpoints to be a little more consistent (like all jump at once), that would be good.  Not a huge deal, but should be improved at some point.


https://github.com/user-attachments/assets/a24ca602-522d-43ae-b3bc-d0a177d53284